listening-comp/backend/performance_tracker.py

```
@@ -0,0 +1,787 @@
+                "sample_size": result[2]
+            }
+        return {
+            "optimal_length": "15-30min",  # Default recommendation
+            "average_accuracy": None,
+            "sample_size": 0
+        }
+    
+    def _calculate_optimal_difficulty(self):
+        """Calculate optimal difficulty level based on performance data."""
+        strengths = self.get_strengths_and_weaknesses()
+        performance = self.get_performance_over_time(days=30)
```

Ruff (0.8.2)

Local variable `performance` is assigned to but never used

Remove assignment to unused variable `performance`

(F841)